soh/src/overlays/actors/ovl_En_GirlA/z_en_girla.c

```diff
@@ -413,12 +474,12 @@ void EnGirlA_InitItem(EnGirlA* this, PlayState* play) {
         osSyncPrintf("引数がおかしいよ(arg_data=%d)！！\n", this->actor.params);
         osSyncPrintf(VT_RST);
         assert((params >= SI_MAX) && (params < 0));
+        //ASSERT(0, "0", "../z_en_girlA.c", 1421); zret assert, currently errors. I assume a macro?
```

Comment:
This assert always fails and it means that this should never be executed, the reason the macro errors is because we don't have that macro yet.